### ISSUE TITLE
[JENKINS-71494] WCAG Operable:2.1 - Keyboard operation and navigation - Result 2. Page areas are not efficiently skippable

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-14#7024]
<p>Finding from the audit attached to Epic -this may have been done by the previous accessibility work, please confirm:</p>
<ul>
	<li><em>In addition to headings, side areas offer a possibility for structural navigation</em></li>
	<li><em>If not all areas of the application are semantically marked as such, they cannot be accessed directly by screen reader or keyboard users (via browser add-on)</em></li>
	<li><em>The breadcrumb and the sidebar are to be marked as navigation areas with the element.</em></li>
	<li><em>To distinguish between the two navigation areas, a meaningful aria label must be assigned, which makes it clear which navigation area is involved. The main content area is to be marked with the element.</em></li>
</ul>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=cristinap">cristinap</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-71494" target="_blank">WCAG Operable:2.1 - Keyboard operation and navigation - Result 2. Page areas are not efficiently skippable</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Finding from the audit attached to Epic -this may have been done by the previous accessibility work, please confirm:</p>
<ul>
	<li><em>In addition to headings, side areas offer a possibility for structural navigation</em></li>
	<li><em>If not all areas of the application are semantically marked as such, they cannot be accessed directly by screen reader or keyboard users (via browser add-on)</em></li>
	<li><em>The breadcrumb and the sidebar are to be marked as navigation areas with the element.</em></li>
	<li><em>To distinguish between the two navigation areas, a meaningful aria label must be assigned, which makes it clear which navigation area is involved. The main content area is to be marked with the element.</em></li>
</ul>
</pre>
</details>
</details>
